projects/infinity_net/cosmos_infinity_net_integration.py

```
@@ -0,0 +1,36 @@
+class CosmosSDKIntegration:
+    def __init__(self):
+        self.cosmos_sdk = CosmosSDK()
```

Ruff (0.8.2)

Undefined name `CosmosSDK`

(F821)